tillerinobot/src/main/java/tillerino/tillerinobot/lang/ChineseSimple.java

```diff
@@ -0,0 +1,264 @@
+
+			return new Message(apiUser.getUserName() + ", " + message);
+		}
+	}
+
+	@Override
+	public String unknownCommand(String command) {
+		return "错误的指令 \"" + command
+				+ "\". 请输入 !help 来查看可用指令!";
+	}
+
+	@Override
+	public String noInformationForMods() {
+		return "抱歉，现在没有这个Mods的信息。";
+	}
+
+	@Override
+	public String malformattedMods(String mods) {
+		return "咦？输入的Mod不对。Mods的缩写包括：DT HR HD HT EZ NC FL SO NF。可以任意组合或者使用其中一个。比如：!with HDHR, !with DTEZ";
+	}
+
+	@Override
+	public String noLastSongInfo() {
+		return "我不记得有给过你歌曲呢...";
+	}
+
+	@Override
+	public String tryWithMods() {
+		return "上Mod玩这张图试试？";
+	}
+
+	@Override
+	public String tryWithMods(List<Mods> mods) {
+		return "加上" + Mods.toShortNamesContinuous(mods) + "试试？";
+	}
+
+	/**
+	 * The user's IRC nick name could not be resolved to an osu user id. The
+	 * message should suggest to contact @Tillerinobot or /u/Tillerino.
+	 * 
+	 * @param exceptionMarker
+	 *            a marker to reference the created log entry. six or eight
+	 *            characters.
+	 * @param name
+	 *            the irc nick which could not be resolved
+	 * @return
+	 */
+	public String unresolvableName(String exceptionMarker, String name) {
+		return "Your name is confusing me. Are you banned? If not, pls check out [https://github.com/Tillerino/Tillerinobot/wiki/How-to-fix-%22confusing-name%22-error this page] on how to resolve it!"
+				+ " if that does not work, pls [https://github.com/Tillerino/Tillerinobot/wiki/Contact contact Tillerino]. (reference "
+				+ exceptionMarker + ")";
+	}
+
+	@Override
+	public String excuseForError() {
+		return "很抱歉，刚刚我没注意听...可以再说一次你想做的事情么？";
+	}
+
+	@Override
+	public String complaint() {
+		return "你的反馈已经发送。Tillerino很快会收到你的反馈。";
+	}
+
+	@Override
+	public Response hug(OsuApiUser apiUser) {
+		return new Message("可以过来一下么？")
+			.then(new Action("抱起 " + apiUser.getUserName()));
+	}
+
+	@Override
+	public String help() {
+		return "嗨！我是谋杀了Tillerino后使用他的账号的机器人。"
+				+ " [https://twitter.com/Tillerinobot TillerinoBot项目]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki 指令]"
+				+ " - [http://ppaddict.tillerino.org/ pp计算]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki/Contact 联系我]";
+	}
+
+	@Override
+	public String faq() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ 常见问题]";
+	}
+
+	@Override
+	public String featureRankRestricted(String feature, int minRank, OsuApiUser user) {
+		return "抱歉，现在 " + feature + " 只能被 " + minRank + "以上的人使用。";
+	}
+
+	@Override
+	public String mixedNomodAndMods() {
+		return "Nomod就是没有mod啦~";
+	}
+
+	@Override
+	public String outOfRecommendations() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ#the-bot-says-its-out-of-recommendations-what-do"
+				+ " 我会把我知道的合适的图都推荐给你。]."
+				+ " 试试别的推荐算法，或者用 !reset 来重置推荐。如果你不确定应该如何使用，请输入 !help 来获取帮助";
+	}
+
+	@Override
+	public String notRanked() {
+		return "Looks like that beatmap is not ranked.";
```

Comment:
No translation? I know this message probably shouldnt be reached anymore since oppai, but just in case :wink: